src/Apps/About2/AboutApp2.tsx

```diff
@@ -16,7 +18,7 @@ import {
   FullBleedHeader,
   FullBleedHeaderOverlay,
 } from "Components/FullBleedHeader"
-import { Masonry } from "Components/Masonry"
+import { scrollIntoView } from "Utils/scrollHelpers"
```

Comment:
Had to use this rather than hook because we're calling the function from outside a react component, and no way to pass in.